charts/incubator/kubernetes-reflector/values.yaml

```
@@ -0,0 +1,80 @@
+image:
+  # TODO: update image to use TC repo
+  repository: emberstack/kubernetes-reflector
+  pullPolicy: IfNotPresent
+  tag: 7.1.210
+
+operator:
+  # TODO: Change to true once support is built in for it in common lib
+  register: false
```

Support in common is only needed for detecting operators, not for registering, registering is safe.

```suggestion
  register: true
```